packages/app-builder-lib/src/targets/FpmTarget.ts

```diff
@@ -292,6 +290,36 @@ export default class FpmTarget extends Target {
     await packager.info.emitArtifactBuildCompleted(info)
   }
 
+  private async executeFpm(target: string, fpmConfiguration: FpmConfiguration, env: any) {
+    const fpmArgs = ["-s", "dir", "--force", "-t", target]
+    if (process.env.FPM_DEBUG === "true") {
+      fpmArgs.push("--debug")
+    }
+    if (log.isDebugEnabled) {
+      fpmArgs.push("--log", "debug")
+    }
+    fpmConfiguration.customDepends?.forEach(it => fpmArgs.push("-d", it))
+    if (target === "deb") {
+      fpmConfiguration.customRecommends?.forEach(it => fpmArgs.push("--deb-recommends", it))
+    }
+
+    fpmArgs.push(...this.configureTargetSpecificOptions(target, fpmConfiguration.compression ?? "xz"))
+    fpmArgs.push(...fpmConfiguration.args)
+
+    const fpmPath = await getFpmPath()
+
+    await exec(fpmPath, fpmArgs, { env }).catch(e => {
+      if (e.message.includes("Need executable 'rpmbuild' to convert dir to rpm")) {
+        const hint = "to build rpm, executable rpmbuild is required, please install rpm package on your system. "
+        if (process.platform === "darwin") {
+          log.error(null, hint + "(brew install rpm)")
+        }
+        log.error(null, hint + "(sudo apt-get install rpm)")
```

Comment:
If it's on Windows, will it prompt you to make a change?

Reply:
`getFpmPath` returns `fpm` on Windows directly from PATH. Based o the Go code, it doesn't seem to hit this error message specifically on Windows